bpaotu/bpaotu/frontend/src/pages/search_page/reducers/types.ts

```diff
@@ -24,6 +24,13 @@ export interface GalaxySubmission {
   error?: string
 }
 
+export interface BlastSubmission {
+  submissionId?: string
```

Comment:
Why is this optional? Don't we always have a submissionId?

Reply:
fixed